api/src/services/survey-service.ts

```diff
@@ -279,8 +297,15 @@ export class SurveyService extends DBService {
     }, []);
   }
 
-  async getSummaryResultId(surveyId: number): Promise<number> {
-    return this.surveyRepository.getSummaryResultId(surveyId);
+  /**
+   * Get a survey summary submission record for a given survey id.
+   *
+   * @param {number} surveyId
+   * @return {*}
+   * @memberof SurveyService
+   */
+  async getSurveySummarySubmission(surveyId: number) {
```

Comment:
nit: would be nice to see the return types here

Reply:
For sure. Was playing around at one point with letting types bubble up from the lowest level repo function. The types do work if you hover over the function or return value, but I agree its not as obvious at a glance anymore which is annoying.

Will update.

Reply:
updated